src/main.rs

```diff
@@ -18,11 +19,11 @@ use gluon::{new_vm, Compiler, Thread, Error, Result};
 use gluon::vm::thread::ThreadInternal;
 #[cfg(not(test))]
 use gluon::vm::Error as VMError;
-#[cfg(not(test))]
-use clap::{Arg, App};
 
 mod repl;
 
+const GLUON_VERSION: &'static str = "0.3.0";
```

Comment:
I don't have a strong opinion on the name nor whether the value should even exist. In my opinion, hard-coding the version in the macro just moves it into the code and obfuscates it. 

It wouldn't be troublesome to change it later, either way.